tests/FakeItEasy.Tests.Approval/ApprovedApi/FakeItEasy.Extensions.ValueTask/net45.approved.txt

```diff
@@ -1,4 +1,5 @@
-﻿[assembly: System.Runtime.InteropServices.ComVisible(false)]
+﻿[assembly: System.Reflection.AssemblyMetadata("RepositoryUrl", "https://github.com/FakeItEasy/FakeItEasy.git")]

```

Comment:
Actually, if we approve this, we *must* use the .NET 5.0 SDK to build, since previous versions don't emit this assembly. I'll update how_to_build.md as well.